src/adres-crab/adres-crab.ts

```diff
@@ -67,6 +67,10 @@ export class AdresCrab {
       this.config.straat.autocompleteType = autocompleteType.Suggest;
     }
 
+    if (this.config.countryId) {
+      this.data.land = { code: this.config.countryId };
+      return;
+    }
     this.data.land = this.data.land || { code: 'BE', naam: 'België' };
```

Comment:
Wordt het land mogelijk nog ergens anders geassigned dan in deze method?
Zoja, ok. Anders misschien oneliner van maken?

```ts
this.data.land = this.config.countryId ? { code: this.config.countryId } : { code: 'BE', naam: 'België' }
```